chaoscenter/authentication/go.mod

```diff
@@ -60,3 +60,7 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/golang-jwt/jwt/v4 => github.com/golang-jwt/jwt/v4 v4.5.0
+
+replace github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.5.2
```

Comment:
Since, the version is already updated above, these replace statements shouldn't be required. Please remove & try.